FlowCal/io.py

```diff
@@ -164,6 +164,12 @@ def read_fcs_text_segment(buf, begin, end, delim=None):
     buf.seek(begin)
     raw = buf.read((end+1)-begin)
 
+    # Some files have been found to include space characters at the end of the
+    # TEXT segment. If the delimiter is not a space character, strip spaces
+    # from the end of the segment.
+    if not delim.isspace():
+        raw = raw.rstrip()
+
```

Comment:
I like this idea more than what I had. I changed it in 5185f7a692852d5bb6ef74ca44095fb30483a420. The same number of files pass/fail.